3.3-apache/Dockerfile

```diff
@@ -28,11 +44,12 @@ RUN apt-get update \
  && sed -i /etc/modsecurity.d/modsecurity.conf \
       -e 's/SecRuleEngine DetectionOnly/SecRuleEngine On/g'
 
-COPY httpd-logging-before-modsec.conf /usr/local/apache2/conf/extra/
-COPY httpd-logging-after-modsec.conf  /usr/local/apache2/conf/extra/
-COPY docker-entrypoint.sh /
+COPY common/modsecurity.conf /etc/modsecurity.d/
+COPY 3.3-apache/conf/*.conf /usr/local/apache2/conf/extra/
+COPY 3.3-apache/docker-entrypoint.sh /
+COPY common/modsec-activate-rules.sh /
 
-EXPOSE 80
+EXPOSE ${PORT}
 EXPOSE 443
```

Comment:
Is this required?

Reply:
if the port is not hard-coded anymore I would say definitely yes